packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfCustomControlLibrary-CSharp/net5.0/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+using System.Windows;
+
+[assembly:ThemeInfo(
+    ResourceDictionaryLocation.None, //where theme specific resource dictionaries are located
+                                     //(used if a resource is not found in the page,
+                                     // or application resource dictionaries)
+    ResourceDictionaryLocation.SourceAssembly //where the generic resource dictionary is located
+                                              //(used if a resource is not found in the page,
+                                              // app, or any theme specific resource dictionaries)
```

Comment:
[nit] Missing space after `//`. 

You could also consider aligning all comments, e.g.
```suggestion
[assembly:ThemeInfo(
    ResourceDictionaryLocation.None,          // where theme specific resource dictionaries are located
                                              // (used if a resource is not found in the page,
                                              // or application resource dictionaries)
    ResourceDictionaryLocation.SourceAssembly // where the generic resource dictionary is located
                                              // (used if a resource is not found in the page,
                                              // app, or any theme specific resource dictionaries)
```


ditto below